src/BenchmarksServer/project.json

```diff
@@ -15,7 +15,6 @@
     "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*",
     "Microsoft.Extensions.CommandLineUtils": "1.0.0-*",
     "System.Diagnostics.Process": "4.1.0-*",
-    "Newtonsoft.Json": "8.0.*",
```

Comment:
This is pinning the version to a lower version than MVC requires, which is what is causing the problem. Floating seems like a better option for this and and our other entry points.